languages/csharp/exercises/concept/user-defined-exceptions/UserDefinedExceptionsTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using Xunit;
+
+public class UserDefinedExceptionsTests
+{
+    [Fact]
+    public void Call_Multiply()
```

Comment:
Maybe describe what is being tested more in detail?

```suggestion
    public void Multiply_with_overflow_throws_calculationexception()
```